deny.toml

```diff
@@ -35,6 +35,10 @@ ignore = [
   # https://rustsec.org/advisories/RUSTSEC-2025-0036
   # `surf` is unmaintained
   "RUSTSEC-2025-0036",
+  # https://rustsec.org/advisories/RUSTSEC-2025-0052
+  # `async-std` has been discontinued
+  # Dependent on via surf
+  "RUSTSEC-2025-0052",
```

Comment:
See also https://github.com/twitch-rs/twitch_oauth2/pull/170, where surf is removed. We should remove it here too.